Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/event/model/Document.java

```diff
@@ -94,6 +104,20 @@ public void setCreationTime(Date creationTime) {
     }
 
     /**
+     *
+     * @return the formattedDate
+     */
+    public String getCreationTimeUiDisplay() {
```

Comment:
This is for to display formatted date for selected document on dialogue headers. p:dialogue component is not supporting to use covertdatetime attribute from UI.